README.md

```diff
@@ -271,6 +273,7 @@ Host types are [those provided by eshost](https://github.com/bterlson/eshost#esh
 | SpiderMonkey | `jsshell`, `spidermonkey`, `sm` |
 | V8 | `d8`, `v8` |
 | XS | `xs` |
+| Serenity LibJS | `serenity-js`, `libjs` |
```

Comment:
This table seems to be alphabetically ordered by "Host Type", might want to keep it that way :)